api/graphql/schema.graphql

```diff
@@ -92,7 +92,9 @@ type User {
     @rename(attribute: "indigenous_declaration_signature")
 
   # Applicant info
-  hasDiploma: Boolean @rename(attribute: "has_diploma")
+  hasDiploma: Boolean
+    @rename(attribute: "has_diploma")
+    @deprecated(reason: "hasDiploma to be replaced")
```

Comment:
The standard is fairly new, but for writing the `reason` for a deprecated graphql entity, do you mind using the following pattern, creating a new issue for the removal, and adding it to the [epic](https://github.com/GCTC-NTGC/gc-digital-talent/issues/5137)?

`oldField` is deprecated. Use `newField` instead. Remove in `#issueNumber`.

REF: https://github.com/GCTC-NTGC/gc-digital-talent/pull/7641#discussion_r1297331161

Reply:
I would've, but it doesn't seem like a replacement has been decided or coming soon
https://github.com/GCTC-NTGC/gc-digital-talent/issues/5976#issuecomment-1679077223